packages/core/admin/admin/src/components/UnstableGuidedTour/Context.tsx

```diff
@@ -84,7 +73,31 @@ function reducer(state: State, action: Action): State {
 
     if (action.type === 'reset_all_tours') {
       draft.enabled = true;
-      draft.tours = initialTourState;
+      /**
+       * TODO: Find a way so this is more dynamic
+       */
+      draft.tours = {
+        contentTypeBuilder: {
+          currentStep: 0,
+          length: 0,
+          isCompleted: false,
+        },
+        contentManager: {
+          currentStep: 0,
+          length: 0,
+          isCompleted: false,
+        },
+        apiTokens: {
+          currentStep: 0,
+          length: 0,
+          isCompleted: false,
+        },
+        strapiCloud: {
+          currentStep: 0,
+          length: 0,
+          isCompleted: false,
+        },
+      };
```

Comment:
I'll refactor this but I think we should just merge it for now to unblock the release QA